docs/developer/jobs.md

```diff
@@ -0,0 +1,67 @@
+# Jobs
+
+This document intents to provide an overview over our jobs running via Prow, GitHub actions and Google Cloud Build.
+
+#### Builds and Tests
```

Comment:
I think we should write somewhere - eventually in the title of this paragraph - that those tests are for the v1alpha4 version (currenty master branch)